lib/src/test/kotlin/com/wire/sdk/service/conversation/ConversationServiceTest.kt

```diff
@@ -185,7 +188,74 @@ class ConversationServiceTest {
             }
         }
 
+    @Test
+    fun whenUpdatingConversationMemberRoleThenUpdatesLocalDatabase() =
```

Comment:
This test is nice but at the same time it's testing mocks basically.
Optionally I would check to add a similar test, but in WireApplicationManagerTest. In that case the backendClient is using Wiremock, but most importantly the appStorage is called for real.
So you actually test that the SQL queries work.

Maybe a new test or just editing one of the current ones where a conversation is created. You can make a member admin, and then call getStoredConversationMembers() and check the results